client/my-sites/stats/stats-notices/index.tsx

```diff
@@ -47,6 +47,12 @@ const ensureOnlyOneNoticeVisible = (
 const NewStatsNotices = ( { siteId, isOdysseyStats, statsPurchaseSuccess }: StatsNoticesProps ) => {
 	const dispatch = useDispatch();
 
+	// find out if a site is commerical or not. handle potential null value as a false
+	const isCommercialRaw = useSelector( ( state ) =>
+		getSiteOption( state, siteId, 'is_commercial' )
+	);
+	const isCommercial = isCommercialRaw === null ? false : Boolean( isCommercialRaw );
```

Comment:
I guess `null` would be treated as `false` anyway. To simplify it, maybe we could just apply `!!` to L52